src/Bridges/vectorizebridge.jl

```diff
@@ -0,0 +1,102 @@
+const LPCone{T} = Union{MOI.EqualTo{T}, MOI.LessThan{T}, MOI.GreaterThan{T}}
+
+vector_set(::MOI.EqualTo) = MOI.Zeros(1)
+vector_set(::MOI.LessThan) = MOI.Nonpositives(1)
+vector_set(::MOI.GreaterThan) = MOI.Nonnegatives(1)
+
+vector_set_type(::Type{<:MOI.EqualTo}) = MOI.Zeros
+vector_set_type(::Type{<:MOI.LessThan}) = MOI.Nonpositives
+vector_set_type(::Type{<:MOI.GreaterThan}) = MOI.Nonnegatives
+
+"""
+    VectorizeBridge{T}
+
+Transforms a constraint `AbstractScalarFunction`-in-`S` where `S <: LPCone` to
+`AbstactVectorFunction`-in-`vector_set_type(S)`.
+"""
+mutable struct VectorizeBridge{T, F, S} <: AbstractBridge
+    vector_constraint::CI{F, S}
+    constant::T
+    set_constant::T # Need to store it as it is added in ConstraintPrimal
+end
+function VectorizeBridge{T, F, S}(model::MOI.ModelLike,
+                                  f::MOI.AbstractScalarFunction,
+                                  set::MOI.AbstractScalarSet) where {T, F, S}
+    set_constant = MOIU.getconstant(set)
+    g = MOIU.operate(-, T, f, set_constant)
+    constant = MOI._constant(g)[1]
```

Comment:
Why do we have both `MOIU.getconstant` and `MOI._constant`? How is a reader supposed to distinguish these?

Reply:
Yes naming are very bad, we need to change these https://github.com/JuliaOpt/MathOptInterface.jl/issues/618